wiki/Ranking_Criteria/Ranking_Criteria_Council/id.md

```diff
@@ -0,0 +1,186 @@
+# Ranking Criteria Council
+
+**Ranking Criteria Council** merupakan kelompok pengguna yang secara khusus dibentuk pada tahun 2016 untuk merombak ulang [Ranking Criteria](/wiki/Ranking_Criteria).
+
+Dahulu kala, terdapat banyak hal pada Ranking Criteria yang tidak terperinci dengan jelas. Para anggota [Quality Assurance Team](/wiki/Modding/Quality_Assurance_Team) (*QAT*) sering kali terpaksa menilai sesuatu hanya berdasarkan pengalaman dan sudut pandang mereka masing-masing. Hal ini memicu banyaknya beatmap yang didiskualifikasi atas alasan yang saling tidak konsisten antar satu sama lain, yang pada akhirnya mendorong pembentukan Ranking Criteria Council agar ke depannya osu! dapat memiliki Ranking Criteria yang terikat pada suatu standar yang jelas.
+
+Para anggota Ranking Criteria Council saling berdiskusi pada [sub-forum Ranking Criteria](https://osu.ppy.sh/community/forums/87) secara tertutup. Anggota-anggota komunitas lainnya dapat melihat hal-hal apa saja yang mereka diskusikan namun tidak dapat berpartisipasi di dalamnya.
+
+## Keanggotaan Ranking Criteria Council
```

Comment:
kalau semisalnya ini dari "Ranking Criteria Council" menjadi, "Dewan Ranking Criteria" bagaimana? karena Council sendiri bisa diartikan menjadi Dewan.

Reply:
don't think this would be a good idea tbh : 

a) specific technical terms are (on most cases) not supposed to be translated

b) the name of the group is literally `Ranking Criteria Council` without any other language equivalent. it's similar to how, for example, NATs are being called `Nomination Assessment Team` regardless of what language you're currently browsing the website in; in fact, forcing the NAT to be locally referred to as `Tim Penilai Nominasi` or the likes of it would be a bad idea for obvious reasons lol